data/Malta/Assembly/ep-popolo-v1.0.json

```diff
@@ -4597,6 +4558,11 @@
     },
     {
       "classification": "party",
+      "id": "party/partit_nazzjonalista",
+      "name": "Partit Nazzjonalista"
```

Comment:
This is the correct version — the existing data is wrong. I've fixed the scraper in https://github.com/everypolitician-scrapers/malta-parliament/commit/33706b0c, so this should sort itself out on the next build.